### DISCUSSION TITLE
Bring back TouchEffect.NativeAnimation

### DISCUSSION THREAD

**Discussion by ziomek64** [CommunityToolkit/Maui#1103] [New Feature Discussions]
So currently I think there is no way to put this wave effect animation on anything we want. We have tap gesture recognizer but that doesn't give an animation.

My use case scenario: Collection View of Frames with images in them. Having that touch effect would give my wave animation only on the image itself and nothing around it.

Is this possible to achieve now? I know there is xamarin toolkit compat but it doesn't work with .net 7 yet from what I see.

**Comment by CliffAgius** [comment]
Closed as per the Community Stand-up discussion here: https://youtu.be/N9wMcBP4jtg?t=2889

**Reply by ziomek64** [reply]
Ms is a joke

**Reply by TheCodeTraveler** [reply]
Hi @ziomek64. I've hidden your comment. I think you'll agree that it is not productive and it reflects poorly on your character.

Every maintainer of `CommunityToolkit.Maui` is a volunteer and we spend our free time on nights and weekends contributing to this project. `CommunityToolkit.Maui` is not owned by Microsoft we do not get any monetary compensation from Microsoft for volunteering on this project. In fact, almost every contributor is a community member, not a Microsoft employee.

We closed this Discussion because it has been open longer than 6 months and has received fewer than 6 upvotes from the community. This Discussion has also not produced any conversations around how it could be implemented. These guidelines are outlined in the YouTube video linked above by Cliff. For reference, you can see the [`Rating View Discussion`](https://github.com/CommunityToolkit/Maui/discussions/1495) that has generated much interested from the community and thus has been approved and will be implemented by a fellow community member.

If you'd like to spearhead the initiative for `TouchEffect.NativeAnimation`, please provide us a detailed design of the API for us to help review, and then you're welcome to write the code and submit a PR, that includes the complete documentation and unit test suite for the new feature.

**Reply by ziomek64** [reply]
@brminnick 
Okay then, community toolkit is not backed by Microsoft but Maui surely is. Why don't somebody answer to people's criticism over here for example : https://github.com/dotnet/maui/discussions/15203

We are very vocal about issues but nobody seems to care.

**Reply by bijington** [reply]
I don't know why you are asking us why the Maui team isn't responding to that massive thread when we aren't part of that team as Brandon pointed out.

I will say that from the bits I have read of that thread, a lot of it is not even close to constructive criticism so I can only imagine how damaging that could be to the team that work so hard on the framework.

I know that the MAUI team works extremely hard and for the size of the team they do amazingly well.
